src/SFA.DAS.EAS.Web/Controllers/EmployerManageApprenticesController.cs

```diff
@@ -408,6 +408,7 @@ public async Task<ActionResult> SubmitChanges(string hashedAccountId, string has
         [HttpGet]
         [Route("{hashedApprenticeshipId}/changes/view", Name = "ViewChanges")]
         [OutputCache(CacheProfile = "NoCache")]
+        [Deprecated]

```

Comment:
should these not redirect to the v2 url if hit? Additionally if so then we agreed to log to show that somewhere is still pointing to old urls.